src/plugins/discover/public/application/components/sidebar/discover_field.tsx

```diff
@@ -133,6 +133,9 @@ export function DiscoverField({
           iconType="plusInCircleFilled"
           className="dscSidebarItem__action"
           onClick={(ev: React.MouseEvent<HTMLButtonElement>) => {
+            if (ev.type === 'click') {
+              ev.currentTarget.focus();
+            }
```

Comment:
@andreadelrio this should prevent Chrome from setting focus to the last focused sidebar element in case of a click

Reply:
Tested this and it seems to solve the issue properly.